### PR TITLE
chore: update paths 🧼

### PR DESCRIPTION
**Pull request by ykhedher**

### CODE REVIEW

**Comment by ykhedher** [comment]
[clean front-end import paths 🧼](https://trello.com/c/O4JoorYm/985-clean-front-end-import-paths-%F0%9F%A7%BC)

**Review by sighphyre — APPROVED** [review]
This looks good to me. I think we should review the front end tests though, and make sure there's a test that touches each file at some point. It would make simple but sweeping changes like this much safer. This does seem to have broken some of the Cypress tests though, can we take a look into that before we merge?

**Review by olav — APPROVED** [review]

**Pull request merged**
